<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>내 컴퓨터 정보</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>컴퓨터 정보</string>
   </property>
   <widget class="QLabel" name="lblCore">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>코어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>CPU(속도)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMemory">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>메모리</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDisk">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>디스크</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>메모리</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCPU">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU(속도)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>코어</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>디스크</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>381</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>네트워크 정보</string>
   </property>
   <widget class="QLabel" name="lblExtraNet">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>외부아이피</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>내부아이피</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNetStat">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>전송상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>전송상태</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInnerNet">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>내부아이피</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-weight: bold;</string>
    </property>
    <property name="text">
     <string>외부아이피</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnRefresh">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
